src/test/beast/IPEndpoint_test.cpp

```diff
@@ -116,7 +116,6 @@ class IPEndpoint_test : public unit_test::suite
       BEAST_EXPECT(v4[2]==0);
       BEAST_EXPECT(v4[3]==1);
 
-      BEAST_EXPECT((!((0xff)<<16)) == 0x00000000);
```

Comment:
My motivation to removing this test is gcc 7 emits the warning about using `<<` in a boolean context, and I believe this test does the same thing as the test below.